File: src/test/java/xyz/oelderoth/runelite/forestry/PluginTest.java
package xyz.oelderoth.runelite.forestry;

import net.runelite.client.RuneLite;
import net.runelite.client.externalplugins.ExternalPluginManager;

public class PluginTest
{
	public static void main(String[] args) throws Exception
	{
		ExternalPluginManager.loadBuiltin(ForestryPlugin.class);
		RuneLite.main(args);
	}
}

File: src/test/kotlin/xyz/oelderoth/runelite/forestry/PluginTest.kt
package xyz.oelderoth.runelite.forestry

import net.runelite.client.RuneLite
import net.runelite.client.externalplugins.ExternalPluginManager

object PluginTest {
    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        ExternalPluginManager.loadBuiltin(ForestryPlugin::class.java)
        RuneLite.main(args)
    }
}